apps/api/src/core/services/config/config.service.ts

```
@@ -5,14 +5,15 @@ interface ConfigServiceOptions<E extends ZodObject<ZodRawShape>, C extends Recor
   config?: C;
 }
 
-export class ConfigService<E extends ZodObject<ZodRawShape>, C extends Record<string, any>> {
+// eslint-disable-next-line @typescript-eslint/ban-types
+export class ConfigService<E extends ZodObject<ZodRawShape>, C extends Record<string, any> = {}> {
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)